### DISCUSSION TITLE
Sonoff TH16  - Console command to get temp reading?

### DISCUSSION THREAD

**Discussion by homestead72** [arendst/Tasmota#14388] [Q&A] [answered]
I have a new Sonoff TH16 flashed with latest version of Tasmota.  I have attached a temp sensor and the temperature and humidity show on the main web page.  At the moment I am just testing with it.

What command on the console should I enter to return the current temperature of the sensor (for creating rules etc)?

Thanks in advance.

Stu

**Comment by barbudor** [comment, marked as answer]
The [Rules](https://tasmota.github.io/docs/Rules/) documentation include multiple exemple on how to trigger a rule on a temperature or humidity measurment

Anything that get published over MQTT, such as the SENSOR message can be used as a trigger for rules

**Reply by sfromis** [reply]
And unless you use the `tele-` prefix, those values are updated about every second (a bit depending on the sensor), not only when you see the TelePeriod payloads being published.

**Reply by homestead72** [reply]
I have not yet got into MQTT and am very much a keen newbie to Tasmota.

I was just interested to see if there was a simple command I could type into the console to return the sensor data back at that point in time?

**Reply by sfromis** [reply]
You do not need to have MQTT connected to get the TelePeriod messages and sensor values to match the messages you see in the console.  You can also get latest values displayed with the command `Sensor 10`, but for rules you need no command to have the sensor values available about every second.

**Reply by homestead72** [reply]
Thanks but typing Sensor 10 in the console does not work for me?

**Reply by sfromis** [reply]
Oops, I meant `Status 10`. But anyway, you can just read what you *already* get in the console, the .../SENSOR payloads.
